src/python/WMCore/Services/RucioConMon/RucioConMon.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+# coding=utf-8
+"""
+Rucio Consistency Monitor Service class developed on top of the native WMCore
+Service APIs providing custom output and handling as necessary for the
+CMS Workload Management system
+"""
+
+from __future__ import division, print_function, absolute_import
+
+from urllib.parse import urlencode
+
+import json
+import logging
+
+from WMCore.Services.Service import Service
+
+
+class RucioConMon(Service):
+    """
+    API for dealing with retrieving information from Rucio Consistency Monitor
+    """
+
+    def __init__(self, url, logger=None, configDict=None):
+        """
+        Init method for the RucioConMon Class
+        """
+        configDict = configDict or {}
+        configDict.setdefault('endpoint', url)
+        configDict.setdefault('cacheduration', 1)  # in hours
+        configDict.setdefault('accept_type', 'application/json')
+        configDict.setdefault('content_type', 'application/json')
+        configDict['logger'] = logger if logger else logging.getLogger()
+        super(RucioConMon, self).__init__(configDict)
+        self['logger'].debug("Initializing RucioConMon with url: %s", self['endpoint'])
+
+    def _getResult(self, uri, callname="", clearCache=True, args=None):
```

Comment:
Shouldn't we actually only clear the cache upon explicit request? If it's been expired, it should anyways make a new call to the web-service.

You might as well improve this docstring a little bit. For instance, what args data type is supposed to be?

Reply:
ok